frontend/package.json

```
@@ -134,16 +134,16 @@
     "postcss": "^8.4.31",
     "qs": "^6.11.0",
     "rimraf": "^3.0.2",
-    "tailwindcss": "^3.3.0",
+    "tailwindcss": "^3.3.3",
```

```suggestion
    "tailwindcss": "^3.3.5",
```

There was an even more current release, so let's update to that.